Umweltzone/src/main/res/values-pt/strings.xml

```diff
@@ -0,0 +1,327 @@
+        A zona abrange as seguintes cidades ou
+        distritos: <xliff:g example="Bochum, Castrop-Rauxel, Essen" id="cities">%1$s</xliff:g> e
+        <xliff:g example="Recklinghausen" id="last_city">%2$s</xliff:g>.
+    </string>
+
+    <string name="city_info_further_information">Mais informações online</string>
+    <string name="city_info_badge_online_title">Formulário de inscrição para adquirir os adesivos</string>
+
+    <string name="city_info_geometry_source_text">Fonte das coordenadas geométricas:
+        <xliff:g example="Geoportal Berlin / Umweltzone Berlin" id="geometrySource">%1$s</xliff:g>
+    </string>
+
+    <string name="city_info_geometry_updated_at_date_format">
+        d de MM de yyyy
+    </string>
+    <string name="city_info_geometry_updated_at_text">
+        Última atualização da geometria da zona: <xliff:g example="6 de Set de 2014" id="updatedAt">%1$s</xliff:g>
+    </string>
+
+    <string name="city_info_allowed_exhaust_emission_standard_in_diesel_prohibition_zone">
+        Apenas veículos que cumprem, pelo menos, com as normas de emissão de gases de Diesel
+        Euro <xliff:g example="6" id="number">%1$d</xliff:g> têm permissão para
+        transitar na zona de proibição de Diesel.
+    </string>
+    <string name="city_info_zone_number_for_residents_since">
+        Para residentes, este regulamento se aplica a partir de <xliff:g example="31 de Mai de 2018" id="date">%1$s</xliff:g>.
+    </string>
+
+    <string name="city_info_zone_number_for_non_residents_since">
+        Para não residentes, este regulamento se aplica a partir de <xliff:g example="31 de Mai de 2018" id="date">%1$s</xliff:g>.
+    </string>
+
+    <string name="city_info_provision_applies_to_vehicles">
+        Esta disposição se aplica a <xliff:g example="carros e veículos de mercadorias pesados" id="vehicles">%1$s</xliff:g>.
+    </string>
+    <string name="city_info_provision_applies_to_vehicles_fragment_cars">carros</string>
+    <string name="city_info_provision_applies_to_vehicles_fragment_hgvs">veículos de mercadorias pesadas</string>
+    <string name="city_info_provision_applies_to_vehicles_fragment_cars_and_hgvs">carros e veículos de mercadorias pesadas</string>
+    <string name="city_info_is_congruent_with_lez">Essa zona é congruente com a zona de baixa emissão.</string>
+
+    <!-- NoCitySelectedFragment -->
+    <string name="no_city_selected_title">Informações da cidade</string>
+    <string name="no_city_selected_status_desc">
+        Informaçoes detalhadas sobre a zona não puderam ser exibidas.
+        Por favor, selecione uma zona antes.
+    </string>
+    <string name="no_city_selected_select_zone">Selecione uma zona</string>
+
+    <!-- FaqActivity -->
+    <string name="faq_activity_title">FAQ</string>
+    <string name="faq_source_url_title_text">Fonte</string>
+
+    <!-- ZoneNotDrawableDialog -->
+    <string name="zone_not_drawable_title">
+        Ajude agora &#8230;
+    </string>
```

Comment:
This acts as a title of a dialog. For some reason the leading space is not removed by the Android resource system. As a result there is a white space indentation before the word. Please change to a one liner:

``` xml
<string name="zone_not_drawable_title">Ajude agora &#8230;</string>
```

Reply:
Forget about this one. I tried it. It doesn't change the indentation.